app/components/navbar.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import Link from "next/link";
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+import GitHubLogoDark from '@/images/logos/github/github-dark.png'
+import TwitterDark from '@/images/logos/twitter/twitter-dark.png'
+
+export default function Navbar() {
+    const pathname = usePathname();
+    const routes = [
+        {
+            text: "docs",
+            href: "/docs"
+        },
+        {
+            text: "benchmarks",
+            href: "/benchmarks"
+        },
+        {
+            text: "examples",
+            href: "/examples"
+        }
+    ];
+    return (
+        <header>
+            <nav className="flex items-center justify-between border-b-2 border-green-700 px-2 py-4 font-semibold text-gray-900" aria-label="Global navigation">
+                <div className="flex items-center gap-4">
+                    <span>undici</span>
+                </div>
+                <div className="flex items-center justify-center gap-4">
+                    {routes.map(route => (
+                        <Link key={route.text} href={route.href} className={pathname === route.href ? "text-green-700": ""}>
```

Comment:
```suggestion
                        <Link key={route.text} href={route.href} className={`capitalize ${pathname === route.href ? "text-green-700": ""}`}>
```

here code snippet to allow capitalize of first letter